apps/backend/app/api/auth/register/route.ts

```
@@ -0,0 +1,245 @@
+import { NextResponse } from "next/server";
+import { Gender, PrismaClient, UserType, Branch} from "@prisma/client";
+import bcrypt from "bcryptjs";
+import { UserError, Status, CreateUserRequest} from "app/types/enums";
+import { sanitize } from "class-sanitizer";
+import { escape as escapeHtml } from "validator";
+
+declare global {
+    var prisma: PrismaClient | undefined;
+}
+
+const prisma = global.prisma || new PrismaClient();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)